src/lib/modules/ipfs/process.js

```diff
@@ -107,6 +93,43 @@ class IPFSProcess extends ProcessWrapper {
     });
   }
 
+  getCorsConfig(cb){
+    let ipfsCorsCmd = `${this.command} config API.HTTPHeaders.Access-Control-Allow-Origin`;
+
+    child_process.exec(ipfsCorsCmd, {silent: true}, (err, stdout, stderr) => {
+      if(err || stderr){
+        err = (err || stderr).toString();
+        return cb(err);
+      }
+      cb(null, stdout);
+    });
+  }
+
+  updateCorsConfig(cb){
+    // update IPFS cors before spawning a daemon (muhaha)
+    let ipfsCorsCmd = `${this.command} config --json API.HTTPHeaders.Access-Control-Allow-Origin "[\\"${this.cors.join('\\", \\"')}\\"]"`;
+    console.trace(`Updating IPFS CORS using command: ${ipfsCorsCmd}`);
+    child_process.exec(ipfsCorsCmd, {silent: true}, (err, stdout, stderr) => {
+      if(err || stderr){
+        err = (err || stderr).toString();
+        return cb(err);
+      }
+      child_process.exec(this.command + ' config --json API.HTTPHeaders.Access-Control-Allow-Credentials "[\\"true\\"]"', {silent: true}, (err, stdout, _stderr) => {
+        if(err || stderr){
+          err = (err || stderr).toString();
```

Comment:
I believe `stderr` has to be `_stderr` here

Reply:
Actually, he should remove the `_` from `_stderr` as it is used.

Reply:
`stderr` comes from the outer try catch block. I do believe we want to use what's injected as `_stderr` here..

Reply:
This should be all sorted out now. Agree that when the variable is **not** used, it should be prefixed with a `_`, to make the linter happy, and to allow access to other params passed in. However, if the variable is used, it should not be prefixed with `_`.